components/webui/common/src/index.ts

```
@@ -61,6 +64,7 @@ interface ServerToClientEvents {
 /**
  * Empty but required by Socket IO.
  */
+// eslint-disable-next-line @typescript-eslint/no-empty-object-type
 interface InterServerEvents {
 }
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)